### DISCUSSION TITLE
App usage analyzer track for WhatsApp

### DISCUSSION THREAD

**Discussion by Goose1225** [community/community#141752] [Accessibility]
Trying to download it onto my phone I can't find it Play store don't have it Galaxy store don't have it I'll mom
app :
App usage analyzer: Tacking forWhatsApp

**Comment by queenofcorgis** [comment]
Hi @Goose1225 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on the Play store or the app. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
